lightning/src/ln/channelmanager.rs

```diff
@@ -924,6 +924,15 @@ impl ClaimablePayment {
 			self.htlcs.iter().map(|htlc| (htlc.prev_hop.channel_id, htlc.prev_hop.htlc_id))
 		)
 	}
+
+	/// Returns the inbound `(channel_id, user_channel_id)` pairs for all HTLCs associated with the payment.
+	///
+	/// Note: The `user_channel_id` will be `None` for HTLCs created using LDK version 0.0.117 or prior.
+	fn get_channel_id_pairs(&self) -> Vec<(ChannelId, Option<u128>)> {
```

Comment:
Thinking about naming. `pairs` seems a bit redundant because the type is also a pair. But don't know anything better. Or just stick with ids?

Not a rust expert, but I believe that `get_` isn't a convention in rust? Could name this (in line with `inbound_payment_id` above) `inbound_channel_ids`?

Reply:
Makes sense! I updated the names to `inbound_channel_ids` in [pr3655.12](https://github.com/shaavan/rust-lightning/commits/pr3655.12)
Thanks a lot for the pointer!